src/main/java/com/ldtteam/blockui/controls/Image.java

```diff
@@ -141,7 +148,10 @@ public static SizeI getImageDimensions(final ResourceLocation resourceLocation)
             }
             catch (final IOException e)
             {
-                Log.getLogger().warn(e);
+                if (!(e instanceof NoSuchFileException || e instanceof FileNotFoundException))
```

Comment:
Normal java way is by putting this is an earlier catch block iirc, and then just add a // Noop comment (or log with debug level as opposed to warn?)